src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.scss

```diff
@@ -228,3 +228,64 @@ p {
   max-width: 600px !important;
   line-height: 2em;
 }
+
+::ng-deep .mat-datepicker-content-container{
```

Comment:
please try not to use ng-deep because it is deprecated. You can assign class name then put it inside scss/styles.scss

Reply:
removed these styles as I am not using it after changes requested by Marti